django_extensions/management/commands/runserver_plus.py

```diff
@@ -454,6 +454,11 @@ def make_environ(self):
             if self.nopin:
                 os.environ['WERKZEUG_DEBUG_PIN'] = 'off'
             handler = DebuggedApplication(handler, True)
+            # Set trusted_hosts (for Werkzeug 3.0.3+)
+            try:
+                handler.trusted_hosts = settings.RUNSERVERPLUS_TRUSTED_HOSTS
```

Comment:
Can't we do something like:
```
trusted_hosts = getattr(settings, 'RUNSERVERPLUS_TRUSTED_HOSTS', None)
if trusted_hosts:
    handler.trusted_hosts = trusted_hosts 
```

That way we avoid avoid the try-except block and are still defensive in not settings it unless the setting actually exists.